pycbc/distributions/joint.py

```diff
@@ -242,6 +242,39 @@ def contains(self, params):
             result = result.item()
         return result
 
+    def __contains__(self, params):
+        """Evaluates whether the given parameters satisfy the boundary
+            conditions, boundaries, and constraints. This method is different
+            from `contains`, that method only check the constraints. When you
+            have a `JointDistribution` instance "joint_dist" and "params", you
+            can use `in` operator to check, such as "params in joint_dist".
+
+        Parameters
+        ----------
+        params : dict, FieldArray, numpy.record, or numpy.ndarray
+            The parameter values to evaluate.
+
+        Returns
+        -------
+        (array of) bool :
+            If params was an array, or if params a dictionary and one or more
+            of the parameters are arrays, will return an array of booleans.
+            Otherwise, a boolean.
+        """
+        params = self.apply_boundary_conditions(**params)
+        result = True
+        for dist in self.distributions:
+            param_name = dist._params[0]
+            contain_array = numpy.ones(len(params[param_name]), dtype=bool)
+            # `__contains__` in `pycbc.distributions.bounded`
```

Comment:
Maybe add a note that we should fix this in the future. I think we don't need to hold up this PR anymore for this particular change, but we should have it so there is method that of the distributions that can except a vector form.